dracut/30ignition/persist-ifcfg.sh

```diff
@@ -29,7 +29,7 @@ persist_ifcfg() {
     # via Ignition, as was supported in OpenShift 4.1.  See
     # https://bugzilla.redhat.com/show_bug.cgi?id=1736875
     ip=$(cmdline_arg ip)
-    if [ "${ip}" = "dhcp" ]; then
+    if [ "${ip}" = "dhcp" ] || [ "${ip}" = "dhcp,dhcp6" ]; then
```

Comment:
Will we ever need _only_ `dhcp6`?

Reply:
It's possible someone is going to try to run this in a pure IPv6 environment with bare metal hosts.  Worth checking that case here, I think.

Reply:
Hmm my understanding is that this logic is just optimizing for the case where the kargs are left untouched from cosa. I just kept the first `[ "${ip}" = "dhcp" ]` bit to make it easier to sync releases of the bits here and there.

There's lots of other ways the user could specify things that we don't catch here, e.g. `ip=<interface>:dhcp` or `ip=dhcp6,dhcp`.

Reply:
That is true. I'm keen to let it go as is. @miabbott :+1: / :-1:?

Reply:
Yup, I misunderstood what was going on here.  So good as is.  :+1:

Reply:
Yep, this is just about detecting the default case, we just need to enumerate things generated by coreos-assembler.